src/asm/rgbasm.5

```diff
@@ -2000,11 +2001,13 @@ POPO
     LD [$FF88], A         ; optimized to use LDH by default
 .Ed
 .Pp
-The options that OPT can modify are currently:
-.Cm b , g , p , h , L ,
+The options that
+.Ic OPT
+can modify are currently:
+.Cm b , g , p , r , h , L ,
 and
 .Cm W .
-The Boolean flag options
+The boolean flag options
```

Comment:
Further above it says "Boolean and/or", not "boolean and/or"; I slightly prefer the capital.